pom.xml

```diff
@@ -162,8 +162,15 @@
                 <artifactId>ossindex-maven-plugin</artifactId>
                 <configuration>
                     <excludeVulnerabilityIds>
-                        <!-- com.google.guava:guava:31.1-jre: Excluded because we assume that the attacker does not have access to the machine. -->
+                        <!-- com.google.guava:guava:31.1-jre required by com.exasol:parquet-io-java:jar:2.0.0 -->
+                        <!-- Rationale for exclusion: We assume that the attacker does not have access to the machine. -->
                         <exclude>sonatype-2020-0926</exclude>
```

Comment:
You can force higher versions of the transitive dependencies in your POM.

Reply:
Yes, tried that, but vulnerability remained, unfortunately.